python/packages/autogen-agentchat/src/autogen_agentchat/agents/_assistant_agent.py

```diff
@@ -635,6 +638,7 @@ def __init__(
         reflect_on_tool_use: bool | None = None,
         tool_call_summary_format: str = "{result}",
         output_content_type: type[BaseModel] | None = None,
+        format_string: str | None = None,
```

Comment:
Let's rename it to `output_content_type_format_string` to indicate the association with `output_content_type`